docs/develop/excel-async-model.md

```diff
@@ -0,0 +1,116 @@
+---
+title: 'Using the async Office Scripts APIs in performance-critical scenarios'
+description: 'A primer on the Office Scripts async APIs and how to use the load/sync pattern to maximize script performance.'
+ms.date: 05/14/2020
+localization_priority: Normal
+---
+
+
+# Using the async Office Scripts APIs in performance-critical scenarios
+
+This article will teach you how to write scripts using the async APIs. These APIs let your script control the data synchronization between the script and the workbook. This gives you maximum control over the network calls to and from the workbook, which are the primary cause of performance issues.
+
+> [!IMPORTANT]
+> The async model is significantly more complicated than the standard Office Scripts APIs. We highly recommend following the guidance in [Improve the performance of your Office Scripts](web-client-performance.md) before switching to the async APIs.
+
+## `main` function
+
+To use the async APIs, your script's `main` function needs to be `async`. It also must take am `Excel.RequestContext` as the first parameter:
```

Comment:
The main function should be the same as the sync version, but it does need the async prefix.

```typescript
async main(workbook: ExcelScript.Workbook) {

}

```

Reply:
The way the async API is added is by using 
```typescript
async main(workbook: ExcelScript.Workbook) {
    Excel.run((context: Excel.RequestContext) => {
        // batch code here
    });
}

Reply:
Thanks for the heads up. I didn't realize this subsection model was ready. I'll redo this article.